### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by amklass11**
# es6 book app

1. modules created 
2. arrow  functions added
3. date added

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+</footer>
+
+<script type = "module" src="scripts.js"></script>
+</body>
+</html>
+
+
+
+
+
```
Comment (brytebee) [inline index.html]: - [ ] **Optional:** Kindly remove extra white lines. It is not professional to leave excess white lines beyond the required one for linters.

File: modules/Time.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  const days = ['Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat'];
+  const curWeekDay = days[today.getDay()];
+  const curDay = today.getDate();
+  const curMonth = months[today.getMonth()];
+  const curYear = today.getFullYear();
+  const date = `${curWeekDay}, ${curDay} ${curMonth} ${curYear}`;
+  document.getElementById('date').innerHTML = date;
+  setTimeout(() => { startTime(); }, 1000);
+}
+export { startTime, checkTime };
```
Comment (brytebee) [inline modules/Time.js:30]: - [x] **Required:** Excellent effort implementing the date/time yourself. 👏👏👏

However, it is a requirement to use `luxon` time library in this milestone. Kindly implement it and request another review.

image

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @amklass11👋,

Good job so far coming so far with the progress you've made! 💪💪

## Highlights
You have done amazing in the following:
- Good set-up and use of modules in the project.✔️
- Great implementation of ES6 syntax.✔️
- Professionally looking project.✔️
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by omar-labana** [comment]
# **Approved ✔️**

Hi @amklass11

Congratulations on completing this project! 👏🏼 🎉

You have correctly implemented all of the requirements📃 correctly and without any errors⚠️ or bugs🪲.

Cheers and good luck in your next milestones! 🍀

Feel free to leave any questions or comments in the PR thread(don't forget to tag me 😄) if something is not 100% clear.

---
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**